app/Http/Requests/Admin/Fields/EditFieldRequest.php

```diff
@@ -29,15 +29,19 @@ public function rules()
         return [
             'name' => 'required|string',
             'type' => 'required|string|in:text,textarea,checkbox',
-            'options' => 'required_if:type,checkbox|json'
+            'options' => 'required_if:type,checkbox|array'
         ];
     }
 
     public function commit(){
         $this->field->name = $this->input('name');
         $this->field->type = $this->input('type');
         if ($this->field->type == 'checkbox') {
-            $this->field->json = $this->input('options');
+            $options = [];
+            foreach ($this->input('options') as $option){
+                $options[] = $option;
+            }
+            $this->field->options = json_encode($options);
```

Comment:
you don't need to json_encode this - you should just use casts (like I did with the kitchen model on the data attribute)